docs/data/gridftp.md

```diff
@@ -83,22 +141,38 @@ This is usually one XML node spread over multiple lines. Note that comments (\#)
 
 #### Selecting a collector host ####
 
-The collector is the central server which logs the GridFTP transfers into a database. There are usually three options:
+The collector is the central server which logs the GridFTP transfers into a database. There are usually two options:
 
 1. **OSG Transfer Collector**: This is the primary collector for transfers in the OSG. Use `CollectorHost="gratia-osg-prod.opensciencegrid.org:80"`.
 1. **OSG-ITB Transfer Collector**: This is the test collector for transfers in the OSG. Use `CollectorHost=" gratia-osg-itb.opensciencegrid.org:80"`.
 
-#### Validation ####
+### Starting the service
```

Comment:
Instead of this separate section, I'd model the `Managing GridFTP` section after [this HTCondor-CE](https://opensciencegrid.org/docs/compute-element/install-htcondor-ce/#using-htcondor-ce) section

Reply:
done